apollo-client/src/main/java/com/ctrip/framework/apollo/util/ApolloHashMapInitialUtil.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.util;
+
+/**
+ * @author vdisk <vdisk@foxmail.com>
+ */
+public class ApolloHashMapInitialUtil {
+
+  /**
+   * calculate initial capacity of hash map {@link java.util.HashSet#HashSet(java.util.Collection)}
+   *
+   * @param expectCapacity expect capacity
+   * @return initial capacity
+   */
+  public static int getInitialCapacity(int expectCapacity) {
```

Comment:
Is it ok to use com.google.common.collect.Sets#newLinkedHashSetWithExpectedSize so that we don't need to maintain this logic in apollo?

Reply:
remove the ApolloHashMapInitialUtil